masterdata/configuration/attributetypes/attributeTypes_program.csv

```diff
@@ -0,0 +1,7 @@
+Uuid,Void/Retire,Entity name,Name,Description,Min occurs,Max occurs,Datatype classname,Datatype config,Preferred handler classname,Handler config,_order:1000
+,,program,ID_Number,ID Number,0,,org.openmrs.customdatatype.datatype.RegexValidatedTextDatatype,[0-9]*,,,,
+,,program,Treatment Date,Treatment Date,0,,org.openmrs.customdatatype.datatype.DateDatatype,,,,,
+,,program,Co-morbidites,Co-morbidites,0,,org.openmrs.customdatatype.datatype.BooleanDatatype,,,,,
+,,program,Doctor-In-Charge,Doctor In Charge,0,,org.openmrs.customdatatype.datatype.FreeTextDatatype,,,,,
+,,program,Patient Stage,Patient Stage,0,,org.openmrs.customdatatype.datatype.ConceptDatatype,56825,,,,
```

Comment:
We should not use concept_id `56825` here, instead use a UUID or concept_name.

Reply:
Yes Updated with UUID instead.As ID might be changed for fresh implementations